File: laramie-tesoreria-webapi/src/domain/entities/archivo.ts
export default class Archivo {

    id: number;
	entidad: string;
	idEntidad: number;
	nombre: string;
	path: string;
	descripcion: string;
	idUsuario: number;
	fecha: Date;

	constructor(
        id: number = 0,
		entidad: string = "",
		idEntidad: number = 0,
		nombre: string = "",
		path: string = "",
		descripcion: string = "",
		idUsuario: number = 0,
		fecha: Date = null
	)
	{
        this.id = id;
		this.entidad = entidad;
		this.idEntidad = idEntidad;
		this.nombre = nombre;
		this.path = path;
		this.descripcion = descripcion;
		this.idUsuario = idUsuario;
		this.fecha = fecha;
	}

	setFromObject = (row) =>
	{
        this.id = row.id ?? 0;
		this.entidad = row.entidad ?? "";
		this.idEntidad = row.idEntidad ?? 0;
		this.nombre = row.nombre ?? "";
		this.path = row.path ?? "";
		this.descripcion = row.descripcion ?? "";
		this.idUsuario = row.idUsuario ?? 0;
		this.fecha = row.fecha ?? null;
	}

}

File: modules/administracion/models/archivo.py
from sqlalchemy import Column, BigInteger, String, DateTime, func

import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from shared.database import Base


class Archivo(Base):
    __tablename__ = "administracion_archivos"

    id = Column(BigInteger, primary_key=True, autoincrement=True)
    entidad = Column(String(50), nullable=False, index=True)
    id_entidad = Column(BigInteger, nullable=False, index=True)
    nombre = Column(String(250), nullable=False)
    path = Column(String(500), nullable=False)
    descripcion = Column(String(500), nullable=True)
    id_usuario = Column(BigInteger, nullable=True)
    fecha = Column(DateTime(timezone=True), server_default=func.now())
